src/ui/public/autoload/modules.js

```diff
@@ -0,0 +1,34 @@
+require('angular');
+require('ui/chrome');
+require('ui/chrome/context');
+require('ui/bind');
+require('ui/bound_to_config_obj');
+require('ui/config');
+require('ui/courier');
+require('ui/debounce');
+require('ui/doc_title');
+require('ui/elastic_textarea');
+require('ui/es');
+require('ui/events');
+require('ui/fancy_forms');
+require('ui/filter_bar');
+require('ui/filter_manager');
+require('ui/index_patterns');
+require('ui/listen');
+require('ui/notify');
+require('ui/parse_query');
+require('ui/persisted_log');
+require('ui/private');
+require('ui/promises');
+require('ui/safe_confirm');
+require('ui/state_management/app_state');
+require('ui/state_management/global_state');
+require('ui/storage');
+require('ui/stringify/register');
+require('ui/styleCompile');
+require('ui/timefilter');
+require('ui/tooltip');
+require('ui/typeahead');
+require('ui/url');
+require('ui/validateDateInterval');
+require('ui/watch_multi');
```

Comment:
Nope, it's the only file that's mostly made up of arbitrary modules which were picked with arbitrary criteria